extra-kernel/dracut/autobuild/overrides/usr/bin/update-initramfs

```diff
@@ -24,9 +24,9 @@ if ! systemd-detect-virt -cq; then
     if [[ -x /usr/bin/grub-mkconfig ]]; then
 	    echo -e "\033[36m**\033[0m\tUpdating GRUB for the new kernel..."
         grub-mkconfig -o /boot/grub/grub.cfg
-    elif [[ -x /usr/bin/systemd-boot-friend && -r /etc/systemd-boot-friend.conf ]]; then
+    elif [[ -x /usr/bin/sbf && -r /etc/systemd-boot-friend.conf ]]; then
```

Comment:
IMO, here may needs `if` instead of `elif` to prevent that if someone uses GRUB 2 and systemd-boot in the same time for some reasons (e.g. use GRUB 2 as  fallback), the kernels don't copy to corresponding  directory yet.

Reply:
Thanks for pointing it out, I will change it to `if` soon after the next update.